style.css

```diff
@@ -0,0 +1,60 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #e5e5e5;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  background-color: white;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 24px;
+}
+
+.header {
+  background-image: url(images/header-shapes\ mobile@2x.svg);
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  padding: 5rem 2rem;
+  border-bottom-left-radius: 10px;
+}
```

Comment:
- [ OPTIONAL ] Please make sure that you give a CSS property of `background-repeat: no-repeat` so that you won't have the background image of this section repeated. :blush: